app/src/main/res/menu/side_menu.xml

```diff
@@ -9,9 +9,13 @@
             android:icon="@drawable/ic_share"
             android:title="Share"
             app:showAsAction="always"/>
+
         <item android:id="@+id/feedback"
             android:icon="@drawable/ic_feedback"
             android:title="Feedback" />
+        <item android:id="@+id/preferences"
+            android:icon="@drawable/ic_baseline_settings_24"
```

Comment:
This drawable asset is missing. So, upload as per your required size.

Reply:
Extremely sorry for this, I am on it to resolve it at earliest.

Reply:
Hi @AyushAgnihotri2025 ,
I have raised a PR of missing files, Kindly verify and merged this PR and the earlier raised PR together.

Reply:
No problem buddy.